python/ntp/commands/run/local/analysis/__init__.py

```diff
@@ -0,0 +1,348 @@
+
+sample = "{dataset}"
+analysisMode = "{mode}"
+    
+settings = getAnalysisSettings(analysisMode)
+if sample in ['TTJets-mcatnlo','TTJets-powheg']:
+    suffixes = {{'TTJets-mcatnlo':'MCatNLO','TTJets-powheg':'POWHEG'}}
+    if settings['custom_file_suffix'] == "":
+        settings['custom_file_suffix'] = suffixes[sample]
+    else:
+        settings['custom_file_suffix'] = suffixes[sample] + '_' + settings['custom_file_suffix']    
+
+
+# Option to process a single ntuple of a sample rather than all of them
+ntupleToProcess = -1
+
+#File for pile-up re-weighting
+PUFile = "{BAT_BASE}/data/" + settings['PUFile']
+PUFile_up = "{BAT_BASE}/data/" + settings['PUFile_up']
+PUFile_down = "{BAT_BASE}/data/" + settings['PUFile_down']
+getMuonScaleFactorsFromFile = True
+getElectronScaleFactorsFromFile = True
+ElectronIdScaleFactorsFile = '{BAT_BASE}/data/ElectronCutBasedID_MediumWP_76X_SF2D.root'
+ElectronIsoScaleFactorsFile = '{BAT_BASE}/data/Elec_SF_Fit_Syst.root'
+ElectronTriggerScaleFactorsFile = '{BAT_BASE}/data/ElectronTriggerEfficiencies.root'
+MuonIdScaleFactorsFile = '{BAT_BASE}/data/MuonID_Z_RunCD_Reco76X_Feb15.root'
+MuonIsoScaleFactorsFile = '{BAT_BASE}/data/MuonIso_Z_RunCD_Reco76X_Feb15.root'
+MuonTriggerScaleFactorsFile = '{BAT_BASE}/data/SingleMuonTrigger_Z_RunCD_Reco76X_Feb15.root'
+getHadronTriggerFromFile = True
+hadronTriggerFile = ''
+ElectronScaleFactorSystematic = settings['ElectronScaleFactorSystematic']
+MuonScaleFactorSystematic = settings['MuonScaleFactorSystematic']
+
+TTbarLikelihoodFile = "{BAT_BASE}/data/" + settings['TTbarLikelihoodFile']
+BTagEfficiencyFile = "{BAT_BASE}/data/" + settings['BTagEfficiencyFile']
+
+#JES Systematic, the +/- number of uncertainties to vary the jets with
+JESsystematic = settings['JESsystematic']
+JetSmearingSystematic = settings['JetSmearingSystematic']
+BTagSystematic = settings['BTagSystematic']
+LightTagSystematic = settings['LightTagSystematic']
+custom_file_suffix = settings['custom_file_suffix']
+
+if ntupleToProcess > 0 :
+    filetype = '*%03d.root' % ntupleToProcess
+    print 'Will only consider ntuple : ',filetype
+    settings['custom_file_suffix'] += str(ntupleToProcess)
+    custom_file_suffix = settings['custom_file_suffix']
+
+inputFiles = inputFiles = [
+{input_files},
+]
+
+print 'Parsed config settings:'
+for setting,value in settings.iteritems():
+    print setting, '=', value
+
+
+#Jet Energy Resolutions files (L7 corrections)
+bJetResoFile = "{BAT_BASE}/data/bJetReso.root"
+lightJetResoFile = "{BAT_BASE}/data/lightJetReso.root"
+
+#Jet Smearing application
+applyJetSmearing = True
+
+#Apply Top Pt reweighting
+applyTopPtReweighting = False
+
+#use HitFit for analysis
+useHitFit = False
+produceFitterASCIIoutput = False
+
+#MET corrections application
+applyMetSysShiftCorr = False
+applyMetType0Corr = False
+
+#relative Path from calling BAT to the TopQuarkAnalysis folder
+TQAFPath = ""
+
+#integrated luminosity the MC simulation will be scaled to
+lumi = 15933 
```

Comment:
Apologies, I thought I replied to this one.
If you are running AT with the ntp commands you would change it in `python/ntp/commands/run/local/analysis/__init__.py `

I have not found an easy way to import the config from AT, modify it and then write the new config file. A limitation on how we implemented the python config in AT.